src/clientSideScene/sceneEntities.ts

```diff
@@ -498,7 +498,6 @@ export class SceneEntities {
       useFakeExecutor: true,
       engineCommandManager: this.engineCommandManager,
       programMemoryOverride,
-      idGenerator: kclManager.execState.idGenerator,
```

Comment:
these were never passed thru in later functions so i just deleted it cc @Irev-Dev

Reply:
Really? It should, here.

https://github.com/KittyCAD/modeling-app/blob/18e1855fa9231e276e1d87530dcae18bdc8f9d75/src/lang/langHelpers.ts#L76-L82

Reply:
all the ones i deleted had fakeExecutor true, and it does not get passed to the fake executor, i did remove from a different place with it set to false, but thats because i pass thru the whole exec state now if the last run was a success.

Reply:
so these are no longer individually needed

Reply:
its actually nice we passing thru the old state now when we can (and its not fake)

Reply:
Wait, this was removed. I added the ID generator so that IDs can be stable across executions. If you select something, a selection is one of these IDs. Are you sure it's okay to remove this?

Reply:
Right, gotcha. That was a recent change that Nick made, to pass through the entire ExecState.

Reply:
nah i did that as a part of this maybe nick and i had conflicting work? but seems like it was only doing memory and idgenerator, i need the whole thing to properly cache, even tho we dont use it all today we will in the future